engines/account/app/services/account/signup_service.rb

```diff
@@ -23,7 +20,7 @@ def create!(email)
     # @return [Account::User]
     def complete!(signup, options)
       ::ActiveRecord::Base.transaction do
-        user = ::Account::User.create(
+        user = ::Account::User.create!(
           email: signup.email,
           username: options[:username],
           password: options[:password],
```

Comment:
Another approach is to rescue the rollback inside the service, without leaking too much information to the controller.

The main goal for now is to make sure that we destroy the signup only if the user has been created.

We'll implement the rescue inside the `SignupService` but not sure if we have time now.